### PR TITLE
PS-7773 Update Audit Log plugin examples (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch ps-5.7-7773
 Changes to be committed:
	modified:   source/management/audit_log_plugin.rst

### CODE REVIEW

File: doc/source/management/audit_log_plugin.rst [hunk 1]
```diff
@@ -6,129 +6,216 @@
 
 Percona Audit Log Plugin provides monitoring and logging of connection and query activity that were performed on specific server. Information about the activity will be stored in the XML log file where each event will have its ``NAME`` field, its own unique ``RECORD_ID`` field and a ``TIMESTAMP`` field. This implementation is alternative to the `MySQL Enterprise Audit Log Plugin <dev.mysql.com/doc/refman/5.7/en/audit-log-plugin.html>`_
 
+Audit logging documents the database usage. You can use the log for troubleshooting. 
+
+
+Installation
+============
+
+Audit Log plugin installed with |Percona Server|, but is not enabled by default. You can verify the plugin is not enabled by running the following commands:
```
Comment (nastena1606) [inline doc/source/management/audit_log_plugin.rst]:
```suggestion
Audit Log plugin is installed with |Percona Server|, but is not enabled by default. You can verify if the plugin is enabled by running the following commands:
```

File: doc/source/management/audit_log_plugin.rst [hunk 2]
```diff
@@ -6,129 +6,216 @@
...
-============
 
-Audit Log plugin is shipped with |Percona Server|, but it is not installed by default. To enable the plugin you must run the following command: 
 
-.. code-block:: mysql
+Log Format
+==========
 
-   INSTALL PLUGIN audit_log SONAME 'audit_log.so';
+The audit log plugin supports four log formats: ``OLD``, ``NEW``, ``JSON``, and ``CSV``. The ``OLD`` and ``NEW`` formats are based on XML. , The ``OLD`` formats the log record properties as XML attributes. The ``NEW``log records are formatted as XML tags. The information logged is the same in all four formats. The log format choice is controlled by :variable:`audit_log_format` variable.
```
Comment (nastena1606) [inline doc/source/management/audit_log_plugin.rst]:
```suggestion
The audit log plugin supports four log formats: ``OLD``, ``NEW``, ``JSON``, and ``CSV``. The ``OLD`` and ``NEW`` formats are based on XML.  The ``OLD`` formats the log record properties as XML attributes. The ``NEW`` log records are formatted as XML tags. The information logged is the same in all four formats. The log format choice is controlled by :variable:`audit_log_format` variable.
```

File: doc/source/management/audit_log_plugin.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/management/audit_log_plugin.rst]: updated.

File: doc/source/management/audit_log_plugin.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/management/audit_log_plugin.rst]: Fixed.

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**